composer.json

```diff
@@ -23,6 +24,11 @@
 			"Automattic\\WooCommerce\\Pinterest\\": "src/"
 		}
 	},
+	"config": {
+		"platform": {
+			"php": "7.3.0"
```

Comment:
In Google Listings and Ads, we have the latest version of PHP `7.3.27` as this value, rather than `7.3.0`. I don't know how much of a difference it makes in practice, but I *think* that we might want to specify a higher version for this value.

Do you have any more information about which option is better, if any?

Reply:
That shouldn't make much of a difference in practice since most of the packages compatible with 7.3.27 are also compatible with 7.3.0. I think it's best to specify the minimum version that we support though, which is 7.3.0. Just for the sake of consistency with our docs. Because we don't mention the minor patch version anywhere.

Reply:
This config just simulates the PHP version when installing or upgrading packages so when for example a dev has PHP 8 installed and tries to add or update a package, Composer still checks for compatibility with PHP 7.3 when installing the package.